### ISSUE TITLE
--location issue

### ISSUE THREAD

**Issue by Arthisulfuthor** [twintproject/twint#787]
Python 3.7
### Command Ran
twint -s "Inondation" --since 2018-01-22 --until 2018-02-07 -o CRP_Inondation.csv --csv

### Description of Issue
Hi, I'm using Twint for research and I'm using --location to obtain geotagged information for making maps. It's been something like a week that using --location makes my terminal bugged while running the same command without --location works perfectly. 

### Environment Details
Mac OS 10.12.6 Terminal

Thank you for your help !

**Comment by YTKong94** [comment]
It's true. I encounter this issue as well. It works smoothly without --location.

**Comment by MonalisaSarewar** [comment]
Yes, facing the same issue. It is been a week since --location is not working in twint.

**Comment by dangdmi** [comment]
Same on Windows if it makes a difference

**Comment by lucalit888** [comment]
I am also experiencing the same issue ... with the location option set to true, NO tweets are scraped

**Comment by jcs82** [comment]
been same for me for last two weeks!

**Comment by Arthisulfuthor** [comment]
Hi, any improvements for anyone ? I'm still stuck with nothing happening while using --location.
Btw, before it breaks, I didn't have any return when I tried to extract locations' informations before 2013. 
Is someone experiencing the same issue ?
Thanks
